pdq/LCOM96b.go

```diff
@@ -0,0 +1,35 @@
+package pdq
+
+import (
+	"fmt"
+)
+
+func CalcLCOM96b(attributesNum int, methodsNum int, methodsPerAttribute []int) (float64, error) {
+	if attributesNum < 0 || methodsNum < 0 {
+		return -1, fmt.Errorf("invalid metric value: attributesNum %v, methodsNum %v; values must not be less than 0", attributesNum, methodsNum)
+	}
+
+	if attributesNum == 0 || methodsNum == 0 {
+		return 0, nil
```

Comment:
if im not mistaken when there are no attributes or methods, LCOM96b makes no sense, but technically the class is as cohesive as possible

should be like
return 1, nil

or..?

Reply:
From my understanding the formula is meant to measure the LACK of cohesion in methods of a class, so then 1 would mean that there no cohesion at all, and 0 that there is full cohesion.

Reply:
okay, it’s not critical, if anything we’ll fix it in the future